services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/BatchGetResultPage.java

```diff
@@ -74,6 +75,17 @@ public static Builder builder() {
                       .collect(Collectors.toList());
     }
 
+    /**
+     * Returns a map of tables and their respective keys that were not processed with the current response. The UnprocessedKeys
+     * value is in the same form as RequestItems, so the value can be provided directly to a subsequent BatchGetItem operation.
+     *
+     * @return unprocessed keys
+     * @see BatchGetItemResponse#unprocessedKeys()
+     */
+    public Map<String, KeysAndAttributes> unprocessedKeys() {
```

Comment:
As we discussed we should remove this as there's no use-case for needing it with pagination and flatmapped iteration support.